R/plot.R

```diff
@@ -1171,11 +1181,10 @@ MSEv_name_explanation_list <- function(explanation_list) {
   names <- make.unique(names, sep = "_")
   names(explanation_list) <- names
 
-  message(paste0(
-    "User provided an `explanation_list` without named explanation objects.\n",
-    "Use the approach names of the explanation objects as the names (with integer ",
-    "suffix for duplicates).\n"
-  ))
+  msg1 <- "User provided an `explanation_list` without named explanation objects."
+  msg2 <- "Use the approach names of the explanation objects as the names (with integer suffix for duplicates)."
+
+  cli::cli_inform(c("i" = msg1, msg2))
```

Comment:
Should one then have "i" in front of both messages?
```suggestion
  cli::cli_inform(c("i" = msg1, "i" = msg2))
```

Reply:
Just visually. No real importance.

Reply:
Using " " for all of msg2 to get indentation.